src/encoded/tests/test_submit.py

```diff
@@ -1292,6 +1294,71 @@ def test_xls_to_json_pedigree_errors(testapp, project, institution):
+            ['Case1_Person1-1', '', 'Fam1', 'Person1-1', 'M', '20', 'year', '', 'proband', 'peripheral blood', 'Specimen1-1', 'N', 'WGS', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Case1_Person1-2', 'Case1', 'Fam1', 'Person1-2', 'F', '50', 'year', '', 'mother', 'peripheral blood', 'Specimen1-2', 'N', 'WGS', '', '', '']], "Row 2", "Row 4"),
+        ([['Unique Analysis ID*:', 'Analysis ID*:', 'Family ID:', 'Individual ID*:', 'Sex*:', 'Age:', 'Age Units:', 'Birth Year:', 'Relation to Proband*:', 'Specimen Type*:', 'Specimen ID*:', 'Report Required*:', 'Test Requested*:', 'Specimen Collection Date:', 'Sequencing Date:', 'Files:'],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Case1_Person1-1', '', 'Fam1', 'Person1-1', 'M', '20', 'year', '', 'proband', 'peripheral blood', 'Specimen1-1', 'N', 'WGS', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Case1_Person1-2', 'Case1', 'Fam1', 'Person1-2', 'F', '50', 'year', '', 'mother', 'peripheral blood', 'Specimen1-2', 'N', 'WGS', '', '', '']], "Row 3", "Row 5"),
+        ([['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Unique Analysis ID*:', 'Analysis ID*:', 'Family ID:', 'Individual ID*:', 'Sex*:', 'Age:', 'Age Units:', 'Birth Year:', 'Relation to Proband*:', 'Specimen Type*:', 'Specimen ID*:', 'Report Required*:', 'Test Requested*:', 'Specimen Collection Date:', 'Sequencing Date:', 'Files:'],
+            ['Case1_Person1-1', '', 'Fam1', 'Person1-1', 'M', '20', 'year', '', 'proband', 'peripheral blood', 'Specimen1-1', 'N', 'WGS', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Case1_Person1-2', 'Case1', 'Fam1', 'Person1-2', 'F', '50', 'year', '', 'mother', 'peripheral blood', 'Specimen1-2', 'N', 'WGS', '', '', '']], "Row 3", "Row 5"),
+        ([['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Unique Analysis ID*:', 'Analysis ID*:', 'Family ID:', 'Individual ID*:', 'Sex*:', 'Age:', 'Age Units:', 'Birth Year:', 'Relation to Proband*:', 'Specimen Type*:', 'Specimen ID*:', 'Report Required*:', 'Test Requested*:', 'Specimen Collection Date:', 'Sequencing Date:', 'Files:'],
+            ['Case1_Person1-1', '', 'Fam1', 'Person1-1', 'M', '20', 'year', '', 'proband', 'peripheral blood', 'Specimen1-1', 'N', 'WGS', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''], 
+            ['Case1_Person1-2', 'Case1', 'Fam1', 'Person1-2', 'F', '50', 'year', '', 'mother', 'peripheral blood', 'Specimen1-2', 'N', 'WGS', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', ''],
+            ['', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '']], "Row 3", "Row 5")
+    ])
```

Comment:
Since you're re-using many of the same rows here, set the rows as constants and then re-use them as needed, either in new constants or in the parametrize call:
```suggestion

BLANK_ROW = ["", "", ...]
HEADER_ROW = ["Unique Analysis ID*", ...]
....
ROWS_WITH_ERROR_IN_ROWS_2_4 = [BLANK_ROW, HEADER_ROW, ...]
@pytest.mark.parametrize('digested_xlsx, expected_error_row_number_1, expected_error_row_number_2', [
        (ROWS_WITH_ERROR_IN_ROWS_2_4, "Row 2", "Row 4"),
        ...
```